internal/persistence/ent/client/package_test.go

```diff
@@ -10,14 +10,26 @@ import (
 	"github.com/go-logr/logr"
 	"github.com/google/go-cmp/cmp/cmpopts"
 	"github.com/google/uuid"
+	"go.artefactual.dev/tools/ref"
 	"gotest.tools/v3/assert"
 
 	"github.com/artefactual-sdps/enduro/internal/datatypes"
 	"github.com/artefactual-sdps/enduro/internal/enums"
 	"github.com/artefactual-sdps/enduro/internal/persistence"
+	entclient "github.com/artefactual-sdps/enduro/internal/persistence/ent/client"
 	"github.com/artefactual-sdps/enduro/internal/persistence/ent/db"
+	"github.com/artefactual-sdps/enduro/internal/timerange"
 )
 
+func nullUUIDToPtr(u uuid.NullUUID) *uuid.UUID {
+	var p *uuid.UUID
+	if u.Valid {
+		p = &u.UUID
+	}
+
+	return p
+}
+
```

Comment:
Only used twice with UUIDs already known to be valid. I's use `&u.UUID` directly in those two cases.

Reply:
Removed.